pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/ImmutableSegmentDataManager.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.core.data.manager.offline;
+package org.apache.pinot.core.data.manager.realtime;
```

Comment:
Immutable segment is for both OFFLINE table and completed segment of real-time table. We should not put it under real-time package

Reply:
`ImmutableSegmentDataManager` is not used inside of offline anymore. Only realtime. So I think it makes sense to be under this package. Certainly doesn't make sense to have it under `offline` anymore.